docs/attachments/attachment.yaml

```diff
@@ -0,0 +1,128 @@
+paths:
+  /attachments:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Attachments
+      summary: Find all attachments.
+      description: Finds and returns an array with a list of all attachments.
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: fileName
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: sort
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: limit
+          schema:
+            type: integer
+          required: false
+        - in: query
+          name: skip
+          schema:
+            type: integer
+          required: false
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#definitions/attachments'
```

Comment:
Change this line to $ref: '#/definitions/attachments'. There should be a slash after the #

Reply:
yeah, already change it